search.php

```diff
@@ -42,8 +38,9 @@
 						 * If you want to overload this in a child theme then include a file
 						 * called content-search.php and that will be used instead.
 						 */
-						get_template_part( 'loop-templates/content', 'search' );
+						get_template_part( 'loop-templates/content', 'archive' );
```

Comment:
Can you add some logic to load `content-archive-podcast` if the current post is a podcast?

I would look into `get_post_type()` to accomplish this

If you want to test by making a podcast and you need help with that, Slack me